ext/RMagick/extconf.rb

```diff
@@ -3,7 +3,12 @@
 
 module RMagick
   class Extconf
-    require 'rmagick/version'
+    if RUBY_VERSION < '1.9.2'
+      require File.expand_path('../../lib/rmagick/version',
+                               File.dirname(__FILE__))
+    else
+      require_relative '../../lib/rmagick/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.